metricbeat/module/kafka/consumergroup/_meta/fields.yml

```diff
@@ -26,10 +27,12 @@
 
     - name: topic
       type: keyword
+      deprecated: true
```

Comment:
Kafka is still beta, so we could remove the old entries. Instead we could have a more detailed explanation in the changelog or the breaking changes.